code/modules/supermatter/supermatter.dm

```diff
@@ -184,6 +184,8 @@
 		if(damage > explosion_point)
 			for(var/mob/living/mob in living_mob_list)
 				if(istype(mob, /mob/living/carbon/human))
+					if(mob.z != src.z)//only make it effect mobs on the current Z level.
```

Comment:
This shouldn't inside the human if, because the rad damage will still be applied.